src/renderer/components/cluster-manager/clusters-menu.tsx

```diff
@@ -149,14 +144,25 @@ export class ClustersMenu extends React.Component<Props> {
             </Droppable>
           </DragDropContext>
         </div>
-        <div className="add-cluster">
-          <Tooltip targetId="add-cluster-icon">
-            Add Cluster
-          </Tooltip>
-          <Icon big material="add" id="add-cluster-icon" disabled={workspace.isManaged} onClick={this.addCluster}/>
-          {newContexts.size > 0 && (
-            <Badge className="counter" label={newContexts.size} tooltip="new"/>
-          )}
+
+        <div className="WorkspaceMenu">
```

Comment:
would seem more natural to put this at the top, above the cluster icons

Reply:
IMO that sounds like a good idea.

Reply:
Just a note that the placement of this icon was already agreed with @miskun .